addons/project/views/project_views.xml

```diff
@@ -1774,7 +1774,7 @@
             <field name="name">All Tasks</field>
             <field name="res_model">project.task</field>
             <field name="view_mode">tree,kanban,form,calendar,pivot,graph,activity</field>
-            <field name="context">{'default_user_ids': [(4, uid)]}</field>
+            <field name="context">{'default_user_ids': [(4, uid)], 'active_test': False}</field>
```

Comment:
But in that case, we will also see all archived tasks, no? 🤔

Reply:
True, my bad, I've moved the `active_test=False` on the field itself in the search of the view.

I am a bit surprised it doesn't work from the get go, because the context is specified on the field, it seems JS side it's not reusing the context as specified on the field. I don't know if this is expected behaviour or an OWL bug, but the JS team isn't responding to my inquiry. :S

Reply:
I've talked with the JS team, my last push is their proposed solution, there is not much we can do, we disabled the active test, but add an explicit domain on active=True for the model `project.task`, so we can search for only active tasks. (active_test context applies globally, and it's currently a limitation from both framework).